tests/Committee/CommitteeManagerTest.php

```diff
@@ -103,6 +104,34 @@ public function testGetNearbyCommittees()
         ], array_keys($this->committeeManager->getNearbyCommittees($coordinates)));
     }
 
+    /**
+     * @dataProvider provideCommitteesByCoordinatesAndCountry
+     */
+    public function testGetCommitteesByCoordinatesAndCountry($adherentUuid, $expectedCount)
+    {
+        $adherent = $this->getAdherent($adherentUuid);
+        $coordinates = new Coordinates($adherent->getLatitude(), $adherent->getLongitude());
+
+        $committees = $this->committeeManager->getCommitteesByCoordinatesAndCountry(
+            $coordinates,
+            $adherent->getCountry(),
+            $adherent->getPostalCode()
+        );
+
+        $this->assertCount($expectedCount, $committees);
+
+        /** @var Committee $committee */
+        foreach ($committees as $committee) {
+            $this->assertSame($adherent->getCountry(), $committee->getCountry());
+        }
+    }
+
+    public function provideCommitteesByCoordinatesAndCountry(): iterable
```

Comment:
The name is not consistent with the provided data. It will be more explicit to name `provideAdherentUuidsAndCommitteesCount`. What do you think?